Source/Constraint.swift

```diff
@@ -362,7 +362,7 @@ public class Constraint {
         // special logic for updating
         if updateExisting {
             // get existing constraints for this view
-            let existingLayoutConstraints = reverse(layoutFrom!.snp_installedLayoutConstraints)
+            let existingLayoutConstraints = Set<LayoutConstraint>(layoutFrom!.snp_installedLayoutConstraints)
```

Comment:
Are we sure that Set is actually using proper equality checks in Swift? The equality check required to compare constraints is an operate overload (See the bottom of the file).

I'm concerned that this wont actually updating existing constraints which are "identical" but different pointers.